04.class/lib/memo_app.js

```diff
@@ -0,0 +1,82 @@
+import { Memo } from "./memo.js";
+import { View } from "./view.js";
+
+export class MemoApp {
```

Comment:
```suggestion
export class App {
```

Memoはなくしてもいいかも
https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name